packages/url_launcher/url_launcher/lib/src/url_launcher_string.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:url_launcher/src/type_conversion.dart';
```

Comment:
nit: relative url

Reply:
Good catch, I noticed and fixed the other file's copy, but missed this one. I wish VS Code weren't so eager to add package-based imports.